torch/fx/experimental/symbolic_shapes.py

```diff
@@ -279,7 +279,7 @@ def parallel_or(*args):
     Evaluate the logical OR of several arguments, avoiding guarding on
     unbacked SymInts if another argument is definitely True.
     """
-    if any(definitely_true(args) for a in args):
+    if any(definitely_true(a) for a in args):
```

Comment:
@ezyang - sneaking this fix in here, assuming this was just a typo in unused code and not something galaxy-brain?

Reply:
oops nice catch